README.md

```diff
@@ -32,73 +31,83 @@ The package needs to be configured with your account's secret key which is
 available in your [Stripe Dashboard][api-keys]. Require it with the key's
 value:
 
+<!-- prettier-ignore -->
 ```js
 const stripe = require('stripe')('sk_test_...');
 
-const customer = await stripe.customers.create({
+stripe.customers.create({
   email: 'customer@example.com',
-});
+})
+  .then(customer => console.log(customer.id))
+  .catch(error => console.error(error));
 ```
 
-Or using ES modules, this looks more like:
+Or using ES modules and `async`/`await`:
 
 ```js
 import Stripe from 'stripe';
-const stripe = Stripe('sk_test_...');
-//…
+const stripe = new Stripe('sk_test_...');
+
+(async () => {
+  const customer = await stripe.customers.create({
+    email: 'customer@example.com',
+  });
+
+  console.log(customer.id);
+})();
 ```
 
-## Initialize with config object
+### Usage with TypeScript
 
-The package can be initialized with several options:
+As of 8.0.0, Stripe maintains types for the latest [API version][api-versions].
 
-```js
-import ProxyAgent from 'https-proxy-agent';
+Import Stripe as a default import (not `* as Stripe`, unlike the DefinitelyTyped version)
+and instantiate it as `new Stripe()` with the latest API version.
 
-const stripe = Stripe('sk_test_...', {
-  apiVersion: '2019-08-08',
-  maxNetworkRetries: 1,
-  httpAgent: new ProxyAgent(process.env.http_proxy),
-  timeout: 1000,
-  host: 'api.example.com',
-  port: 123,
-  telemetry: true,
+```ts
+import Stripe from 'stripe';
+const stripe = new Stripe('sk_test_...', {
+  apiVersion: '2019-12-03',
```

Comment:
Let's make sure our [release script](https://git.corp.stripe.com/stripe-internal/bindings/blob/6bfe411916735b0b0938f734c76cae9bfab01f81/lib/bindings/node.rb#L12-L23) updates this automatically.